src/lib.rs

```diff
@@ -55,6 +56,18 @@ impl FixedBitSet
         }
     }
 
+    /// Shrink capacity to **bits**.  If **bits** is greater than length, has no effect.
+    pub fn shrink(&mut self, bits: usize) {
```

Comment:
Can we skip making this method public? It doesn't handle the zeroing of bits in the last block, past `bits` entirely, so it's not entirely logically consistent as a public API